EIPS/eip-2309.md

```diff
@@ -51,32 +52,20 @@ RFC 2119.
 
 **ERC-721 compliant contracts MAY implement this Consecutive Transfer Extension to provide a standard event to be emitted at the time of creation, burn, or transfer of one or many consecutive tokens**
 
+The address executing the transaction **MUST** own all the tokens within the range of `fromTokenId` and `toTokenId`, or **MUST** be an approved operator to act on the owners behalf.
+
+The `fromTokenId` and `toTokenId` **MUST** be a consecutive range of tokens IDs.
+
+When minting/creating tokens, the `fromAddress` argument **MUST** be set to `0x0` (i.e. zero address).
+
+When burning/destroying tokens, the `toAddress` argument **MUST** be set to `0x0` (i.e. zero address).
+
+When emitting the ConsecutiveTransfer event the Transfer event **MUST NOT** be emitted
+
+Contracts that implement the `ConsecutiveTransfer` event **MAY** still use the original `Transfer` event, however when emitting the `ConsecutiveTransfer` event the `Transfer` event **MUST NOT** be emitted. 
+
 ```solidity
-pragma solidity ^0.5.10;
-
-/**
-  @title ERC-2309: ERC-721 Consecutive Transfer Extension
-  @dev https://github.com/ethereum/EIPs/issues/2309
- */
-
-interface ERC721ConsecutiveTransfer /* is ERC721 */ {
-  /**
-    @notice This event is emitted when ownership of a consecutive batch of tokens changes by any mechanism.
-    This includes minting, transferring, and burning.
-
-    @dev The address executing the transaction MUST own all the tokens within the range of
-    fromTokenId and toTokenId, or MUST be an approved operator to act on the owners behalf.
-    The fromTokenId and toTokenId MUST be a consecutive range of tokens IDs.
-    When minting/creating tokens, the `fromAddress` argument MUST be set to `0x0` (i.e. zero address).
-    When burning/destroying tokens, the `toAddress` argument MUST be set to `0x0` (i.e. zero address).
-    When emitting the ConsecutiveTransfer event the Transfer event MUST NOT be emitted
-
-    @param fromTokenId The token ID that begins the batch of tokens being transferred
-    @param toTokenId The token ID that ends the batch of tokens being transferred
-    @param fromAddress The address transferring ownership of the specified range of tokens
-    @param toAddress The address receiving ownership of the specified range of tokens.
-  */
-  event ConsecutiveTransfer(uint indexed fromTokenId, uint toTokenId, address indexed fromAddress, address indexed toAddress);
+event ConsecutiveTransfer(uint indexed fromTokenId, uint toTokenId, address indexed fromAddress, address indexed toAddress);
 }
```

Comment:
```suggestion
 event ConsecutiveTransfer(uint indexed fromTokenId, uint toTokenId, address indexed fromAddress, address indexed toAddress);
```

Reply:
Recommend using `uint256` for increased clarity, in case future readers of this specification aren't familiar with Solidity or future versions of Solidity change the width of `uint`.
```suggestion
 event ConsecutiveTransfer(uint256 indexed fromTokenId, uint256 toTokenId, address indexed fromAddress, address indexed toAddress);
```

Reply:
Good call :+1: Updated